docs/source/keylime/index.rst

```diff
@@ -0,0 +1,97 @@
+Keylime
+=======
+
+ESI provides a guide to use `keylime`_ outside Ironic for run-time attestation with `IMA`_.
```

Comment:
I think 'keylime' should be capitalized.

Reply:
ack